packages/react-composites/src/composites/ChatComposite/file-sharing/FileDownloadHandler.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { FileMetadata } from './FileUpload';
+
+/**
+ * @beta
+ *
+ * A callback function for handling file downloads.
+ * The function needs to return a promise that resolves to a file download URL.
+ * If the promise is rejected, the {@link Error.message} will be used to display an error message to the user.
+ *
+ * Sample Usage:
```

Comment:
Does it make more sense to have this file sharing usage information as a .md file in the docs directory?

Reply:
I think there is a lot of value in having descriptive usage eample as part of the docstrings.
We can identify a place to put all of the file sharing documentaion in a single place.
.md file would be a good place to start